packages/core/src/App/Containers/RealAccountSignup/account-wizard.jsx

```diff
@@ -69,42 +69,41 @@ class AccountWizard extends React.Component {
         this.props.fetchStatesList();
         const { cancel, promise } = makeCancellablePromise(this.props.fetchResidenceList());
         this.cancel = cancel;
+        const { cancel: cancelFinancialAssessment, promise: financial_assessment_promise } = makeCancellablePromise(
```

Comment:
why do we need to call the same thing as line :70?

Reply:
They are 2 different promises. Not the same.